packages/utils/src/http.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosError } from 'axios'
+
+export class HttpError {
+  constructor(public msg: string, public httpStatus: number) {}
+}
+
+export type HttpConfig = {
+  timeout_seconds: number
+  max_redirects: number
+}
+
+export async function post(url: string, json_data: string, config: HttpConfig): Promise<string> {
+  try {
+    let response = await axios.post(url, json_data, {
+      timeout: config.timeout_seconds * 1000,
+      maxRedirects: config.max_redirects,
+      headers: {
+        'Accept-Encoding': 'gzip, compress, deflate',
+        Accept: 'application/json',
```

Comment:
Should the `Accept` key also be in single quotes?

Reply:
This is some weirdness of `make fmt`